oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -21,9 +21,7 @@ package oak.session.noninteractive.v1;
 option java_multiple_files = true;
 option java_package = "oak.session.noninteractive.v1";
 
-// AttestationEvidence contains the information that the untrusted launcher provides to the client
-// in response to its request for the enclave's public key(s). The name is chosen to match the RATS
-// terminology (see https://www.rfc-editor.org/rfc/rfc9334.html#name-evidence).
+// AttestationEvidence contains the evidence provided by the enclave.
 message AttestationEvidence {
```

Comment:
I think this one is the Evidence from RATS: https://www.rfc-editor.org/rfc/rfc9334.html#name-evidence (could you please add this link to the comments) ?

Reply:
Done.